.circleci/config.yml

```diff
@@ -103,14 +103,22 @@ jobs:
           when: always
           command: |
             . venv/bin/activate
+            rm -rf dist  # make sure it's clean
             python setup.py sdist bdist_wheel
+            mkdir nevergrad-build
+            mv dist nevergrad-build/
```

Comment:
(btw if you copy-pasted the script from HiPlot: I used a different `dist` folder because TypeScript will generate JS files in `dist` as well, but it might not be necessary here)

Reply:
I considered removing it, but I needed a root (nevergrad-build) and a path (dist), so this was the simplest way I found anyway